src/CONST.ts

```diff
@@ -4,6 +4,11 @@ import * as KeyCommand from 'react-native-key-command';
 import * as Url from './libs/Url';
 import SCREENS from './SCREENS';
 
+// Creating a default array and object this way because objects ({}) and arrays ([]) are not stable types.
+// Freezing the array ensures that it cannot be unintentionally modified.
+const EMPTY_ARRAY = Object.freeze([]);
+const EMPTY_OBJECT = Object.freeze({});
```

Comment:
I see similar definitions in https://github.com/Expensify/App/blob/main/src/hooks/useArrowKeyFocusManager.js and https://github.com/Expensify/App/blob/main/src/hooks/useKeyboardShortcut.js as well, let's clean those up to use these new constants.

Could you also add a comment explaining where/why this is useful?